engine/executor/prom_range_vector_transform_test.go

```diff
@@ -0,0 +1,340 @@
+/*
+Copyright 2024 Huawei Cloud Computing Technologies Co., Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package executor_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/openGemini/openGemini/engine/executor"
+	"github.com/openGemini/openGemini/lib/util/lifted/influx/influxql"
+	"github.com/openGemini/openGemini/lib/util/lifted/influx/query"
+)
+
+func TestNewPromRangeVectorTransformErr(t *testing.T) {
+	call := &influxql.PromSubCall{
+		Name:      "unkown_prom",
+		Interval:  int64(2 * time.Millisecond),
+		StartTime: int64(2 * time.Millisecond),
+		EndTime:   int64(4 * time.Millisecond),
+		Range:     3 * time.Millisecond,
+		Offset:    0,
+	}
+	rowDataType := buildPromBinOpOutputRowDataType()
+	schema := buildPromSubquerySchema()
+	if _, err := executor.NewPromRangeVectorTransform(rowDataType, rowDataType, schema, call); err == nil {
```

Comment:
Why condition `error == nil`  cause `t.Fatal`

Reply:
"unkown_prom" function name should return error.